exams/signals_test.py

```diff
@@ -95,11 +106,30 @@ def test_update_exam_authorization_final_grade(self):
 
     def test_update_exam_authorization_cached_enrollment(self):
         """
-        Verify that update_exam_authorization_final_grade is called when a CachedEnrollment saves
+        Test that on user enroll it create his exam profile
         """
-        # assert Exam Authorization and profile created. CachedEnrollmentFactory triggered the signal.
-        self.assertTrue(ExamProfile.objects.filter(profile=self.profile).exists())
-        self.assertTrue(ExamAuthorization.objects.filter(
-            user=self.profile.user,
-            course=self.course_run.course
-        ).exists())
+        create_order(self.profile.user, self.course_run)
+        CachedEnrollmentFactory.create(user=self.profile.user, course_run=self.course_run)
```

Comment:
It's worth asserting before this that the `ExamProfile` doesn't exist before this because otherwise you're not asserting that it was actually the signal for `CachedEnrollment` that created it.